utils/game-utils.js

```diff
@@ -5,7 +5,20 @@ const getPlayerPnumByUsername = (players, username) => {
 
 const pNumIsValid = (pNum) => [0, 1].includes(pNum);
 
+const getCardName = (card) => {
+  const str_rank =
+    {
+      1: 'A',
+      11: 'J',
+      12: 'Q',
+      13: 'K',
+    }[card.rank] ?? card.rank;
+  const str_suit = ['♣️', '♦️', '♥️', '♠️'][card.suit];
+  return str_rank + str_suit;
+};
```

Comment:
Wondering if this is a good place for this to live, or if this should be in `gameService` and should be called in the api files as `gameService.getCardName(card)`

Reply:
I could go either way. Given that it tends to be called inside of string formatting, it's at least shorter when consumed this way, so I think it's okay